.ai/tools/mcp-vector-server/src/s2c_mcp_server/server.py

```
@@ -0,0 +1,489 @@
+        bounds=bounds,
+        path_count=path_count,
+        used_colors=colors,
+        source_path=str(path),
+    )
+
+
+def _parse_avg(path: Path) -> VectorSummary:
+    """Parse Android Vector Drawable XML (`.xml`) into `VectorSummary`."""
+
+    xml_text = path.read_text(encoding="utf-8")
+    root = et.fromstring(xml_text)
```

Ruff (0.15.2)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)